modules/@apostrophecms/image/ui/apos/components/AposMediaManager.vue

```diff
@@ -397,13 +406,33 @@ export default {
       }
       this.currentPage = currentPage;
       this.totalPages = totalPages;
+      // Do not perform extra work if not needed
+      const skipIds = options.loadMoreIfSkipped && !options.loadMore
+        ? new Set(this.items.map(item => item._id))
+        : new Set();
+
+      let skipped = 0;
```

Comment:
This number is never used right? So it acts like a boolean. I'm not sure how we now how many items we skip then, maybe we just load until no duplicate?

Reply:
I originally had some additional logic (computations involving skipped records). I don't think it's a bad thing to keep track of the count. I'm changing the handler to return the skipped count.